src/types.rs

```diff
@@ -1257,6 +1274,11 @@ impl Term {
     /// let t = parse_term(&mut sig, "A(B x_)").expect("parse of A(B x_)");
     ///
     /// assert_eq!(t.display(&sig), "A(B x_)".to_string());
```

Comment:
This first test might not be necessary given the term you show in the second test. If you agree, please remove.